### PR TITLE
Adds analytics enable / disable based on user cookie acceptance

### PR DESCRIPTION
**Pull request by a6software**
Request to merge in this functionality:

https://github.com/boxuk/peake-wp-content/blob/develop/client-mu-plugins/gdpr/class-gdpr-cookie-tracking.php

### CODE REVIEW

**Review by jmorgan74 — APPROVED** [review]

File: public/class-gdpr-cookie-tracking.php [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	 */
+	public function remove_tracking_scripts() : void {
+		add_filter( 'jetpack_honor_dnt_header_for_stats', '__return_true' );
+
+		/**
+		 * First party
+		 * Disable Google Tag Manager
+		 * Enabled in wp-content/themes/peake/inc/google-tag-manager.php
+		 */
+		remove_action( 'wp_head', 'Peake\Theme\Inc\render_tagmanager_head' );
```
Comment (jenkoian) [inline public/class-gdpr-cookie-tracking.php]: Peake namespace shouldn't be referenced in the plugin?

Comment (a6software) [reply public/class-gdpr-cookie-tracking.php]: Yeah, you're absolutely right. Replaced with a `do_action`, which will be hooked into from Peake.

Corresponding Peake PR - https://github.com/boxuk/peake-wp-content/pull/765

Comment (jenkoian) [reply public/class-gdpr-cookie-tracking.php]: Lovely!

**Review by jenkoian — APPROVED** [review]

**Comment by a6software** [comment]
Latest force push to resolve merge conflict with GDPR extra logging branch.

**Pull request merged**